.github/workflows/linux_meson.yml

```diff
@@ -102,3 +102,111 @@ jobs:
       run: |
         export OMP_NUM_THREADS=2
         python dev.py -n -j 2
+
+  prerelease_deps_coverage_64bit_blas:
+    name: Prerelease deps and Blas 64bit
+    # If using act to run CI locally the github object does not exist and the usual skipping should not be enforced
+    if: "github.repository == 'scipy/scipy' || github.repository == ''"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+
+    - name: Setup Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Ubuntu dependencies
+      run: |
+        # NOTE: not the same OpenBLAS version as in upstream CI (I'm being lazy here)
+        sudo apt-get update
+        sudo apt-get install -y libopenblas-dev libatlas-base-dev liblapack-dev gfortran libgmp-dev libmpfr-dev libsuitesparse-dev ccache libmpc-dev
+
+    - name: Caching Python dependencies
+      uses: actions/cache@v3
+      id: cache
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip
+
+    - name: Install Python packages
+      run: |
+        python -m pip install "setuptools<60.0" wheel ninja meson==0.60.3
+        python -m pip install --pre --upgrade --timeout=60 -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
+        python -m pip install --pre --upgrade --timeout=60 cython pytest pytest-xdist pybind11 mpmath gmpy2 pythran threadpoolctl
+        python -m pip install matplotlib scikit-umfpack scikit-sparse
+        python -m pip uninstall -y scipy
```

Comment:
Why do we need to do this? Do we install SciPy after building?

Reply:
Yes matplotlib is installing scipy and making the rest fail since there are 2 SciPy in the env.

Reply:
This looks a little odd indeed (and may start failing if the pip resolver gets smarter at some point). But it's hard to avoid right now. So fine to leave as is; in the future we can put the built SciPy in `site-packages` and install the packages that depend on SciPy only after that.